www/src/content/docs/docs/get-started.mdx

```diff
@@ -1,86 +1,319 @@
 ---
 title: Get Started
-description: How to get started with Fiberplane Studio
+description: How to get started with Fiberplane Playground
 ---
 
 import PackageManagers from "@/components/PackageManagers.astro";
 import Platforms from "@/components/Platforms.astro";
 import { Aside, Steps, LinkCard } from "@astrojs/starlight/components";
-import getStartedHonoRawCode from "./get-started-cf.ts.example?raw";
-import getStartedNodeRawCode from "./get-started-node.ts.example?raw";
 
-Fiberplane Studio is a local debugging companion designed to help you to build, test, and develop [Hono](https://hono.dev) APIs.
+Fiberplane is a [Hono](https://hono.dev) middleware that provides a built-in Playground UI for exploring and testing Hono service APIs. It leverages [OpenAPI](https://www.openapis.org/what-is-openapi) to deliver interactive API documentation, allowing users to send requests with generated payloads through the UI.
 
-<Steps>
-1. ### Set up a Hono project
+### 1. Set up a Hono project
 
-    You can set up a basic Hono project via your terminal:
+You can set up a basic Hono project via your terminal:
 
-    <PackageManagers type="create" pkg="hono@latest" args="my-app" />
+<PackageManagers type="create" pkg="hono@latest" args="my-app" />
 
-    If you want a project that includes an ORM, a database, and is ready to deploy, we recommend using [HONC 🪿](https://honc.dev/):
+If you want a project that includes an ORM, a database, the option to include Hono Zod OpenAPI (see step 2), Fiberplane middleware (see step 3) and is ready to deploy as a Cloudflare Worker, we recommend using [HONC 🪿](https://honc.dev/):
 
-    <PackageManagers type="create" pkg="honc-app@latest" />
+<PackageManagers type="create" pkg="honc-app@latest" />
 
-2.  ### Add the Fiberplane Studio client
+### 2. Add an OpenAPI spec to Hono
 
-    Install the Studio client library in your Hono project. The client library will report traces to the local Fiberplane Studio and give you rich application context while you're developing.
+The Fiberplane Playground Hono middleware works with an OpenAPI specification. There are two packages that help you define OpenAPI specs in your Hono application.
+[Zod OpenAPI Hono](https://github.com/honojs/middleware/tree/main/packages/zod-openapi) helps you design OpenAPI specs from the start, ensuring all definitions align with the extended OpenAPI class.
+[Hono OpenAPI](https://hono.dev/examples/hono-openapi) generates specs from existing validation schemas, allowing for incremental spec building.
 
-    <PackageManagers pkg="@fiberplane/hono-otel" />
+<Aside type="note">
```

Comment:
probably needs a link to documentation here, or a snippet for how to use `createOpenAPISpec`